README.md

```diff
@@ -247,6 +247,35 @@ APIs!**
 - [BSC-Deploy](https://github.com/bnb-chain/node-deploy/): deploy tool for setting up both BNB Beacon Chain, BNB Smart Chain and the cross chain infrastructure between them.
 - [BSC-Docker](https://github.com/bnb-chain/bsc-docker): deploy tool for setting up local BSC cluster in container.
 
+
+## Running a bootnode
+
+Bootnodes are full nodes that are not behind a NAT. When you start up a node it should log your enode, which is a public identifier that others can use to connect to your node. 
```

Comment:
bootnodes are not full nodes, right? they are just nodes running the Disv4/5 protocol, it will not keep a copy of the blocks, so they are very slim, right?